### PR TITLE
Fix typos

### PR DESCRIPTION
**Pull request by kianmeng**
Found via:
- `codespell -S icons,.git -L cristal,projet,froms,fo,te,theri,couldn,ue,openin,nd`
- `typos --format brief`

### CODE REVIEW

File: src/remote.ml [hunk 1]
```diff
@@ -1581,20 +1581,20 @@ let halfduplex =
...
 
-let negociateFlowControlLocal conn () =
+let negotiateFlowControlLocal conn () =
   disableFlowControl conn.outputQueue;
   Lwt.return false
 
-let negociateFlowControlRemote =
-  registerServerCmd "negociateFlowControl" Umarshal.unit Umarshal.bool negociateFlowControlLocal
+let negotiateFlowControlRemote =
+  registerServerCmd "negotiateFlowControl" Umarshal.unit Umarshal.bool negotiateFlowControlLocal
```
Comment (tleedjarv) [inline src/remote.ml:1589]: This one is a no-go. The string argument is part of the RPC API and must not be changed.

**Comment by tleedjarv** [comment]
Overall, I'm a bit hesitant towards changing actual code under the umbrella of "fixing typos".

**Comment by gdt** [comment]
I am hesitant too, although I am sympathetic to fixing where it is not problematic.   I don't really have cycles to dig in hard right now.

@kianmeng Could you amend (or new, whatever) to just files for human reading?  That seems mergeable much more easily and we'll get half done, arguably the more important half.

**Comment by kianmeng** [comment]
@gdt Noted. I've reverted changes to the source code and keep only those in comments and documentation. Let me know if I miss out anything.

**Comment by gdt** [comment]
Looks good to me, but would appreciate someone else reviewing too.

I approved running the CI workflow; obviously that has to pass but I am pretty sure it will.

**Review by tleedjarv — APPROVED** [review]
Looks ok to me too.

**Comment by kianmeng** [comment]
:partying_face: :partying_face: :partying_face: :partying_face: :partying_face:

**Pull request merged**